src/transformState.js

```diff
@@ -62,7 +62,27 @@
  * @param {Object[]} transforms
  */
 function transformState(state, transforms) {
-  // write code here
+  for (const obj of transforms) {
```

Comment:
for (const { operation, properties } of transforms)

Reply:
obj?

Reply:
fixed!